### ISSUE TITLE
Contract ID documentation missing

### ISSUE THREAD

**Issue by mayhem** [egorFiNE/ropstenfaucet#2]
There is no documentation on how to generate the CONTRACT_ID that is required in the .env file. If you can give me some short tips on how to run that (I'm guessing that is in the hh dir?) I'd be happy to write some docs to help others.

**Comment by egorFiNE** [comment]
Oh yes. That's the sending contract. I have updated the documentation and added a bit about CONTRACT. Thanks for notifying me! 

If you feel like docs is missing on some points - please send PR, I will be happy to merge

**Comment by mayhem** [comment]
I'd be happy to send a PR when I get it to work. You tips on hardhat pointed me in the right direction -- I was able to build the contract and then deploy it, but it produces a contract that shows a balance that is not mine. Clearly I missed some config options somewhere.

Here is what I did:

- Update PRIVATE_KEY in hardhat.config.js 
- Update the infura url to use my own

Then run:
```
npx hardhat compile
npx hardhat run scripts/deploy.js
```

And it produces 0x5FbDB2315678afecb367f032d93F642f64180aa3 , which has transaction on it that are not mine:

https://ropsten.etherscan.io/address/0x5FbDB2315678afecb367f032d93F642f64180aa3

What else needs changing?
